inference/core/interfaces/http/http_api.py

```diff
@@ -1414,6 +1418,28 @@ def validate_workflow(
                 )
                 return WorkflowValidationStatus(status="ok")
 
+        @app.post(
```

Comment:
should we still have an env var to disable this also?
Like in some cases we might not want to expose it (v1 hosted, internal service, etc)?